packages/myst-transforms/src/links/plugin.ts

```diff
@@ -20,7 +20,7 @@ export function formatLinkText(link: Link) {
   if (link.children?.length !== 1 || link.children[0].type !== 'text') return;
   const url = link.children[0].value;
   // Add an exception for wiki transforms links.
-  if (url.length < 20 || url.match(/\s/) || url.startsWith('wiki:')) return;
```

Comment:
I think this should stay.

Reply:
The other changes in this PR that generalise link formatting should ensure that this function is only called when its required (i.e., not for wiki links)